wardrobe/kitties/src/lib.rs

```diff
@@ -146,6 +182,12 @@ impl Default for Parent {
 )]
 pub struct KittyDNA(pub H256);
 
+/// Kitty data contains basic information such as below:
+/// parent: 1 mom kitty and 1 dad kitty.
+/// free_breedings: Free breeding allowed on a kitty.
+/// dna: It's unique per kitty.
+/// num_breedings: Number of free breedings remaining.
+/// name: Name of kitty.
```

Comment:
From this comment, I can't understand how `free_breedings` and `num_breedings` work, and what's the difference between them.

Reply:
Ok, I will update it as below :
1. Free breeding allowed on a kitty. -> Maximum free breeding allowed for a kitty.
2. Number of free breedings remaining -> Current count of remaining free breedings.

Earlier It was without any comments, So I decided to add it.